### PR TITLE
Use bundles 0.2.0 - added testing bundles.

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- use helm-broker bundles 0.2.0
- adjust GCP broker installation documentation

**Related issue(s)**

### CODE REVIEW

File: docs/service-brokers/docs/003-overview-gcp-broker.md [hunk 1]
```diff
@@ -9,11 +9,6 @@ Google Cloud Platform (GCP) Service Broker is an implementation of the [Open Ser
 
 Kyma provides Namespace-scoped GCP Service Broker. In each Namespace, you can configure the GCP Broker against different Google Cloud Platforms. Install the GCP Service Broker by provisioning the **Google Cloud Platform Service Broker Provider** class provided by the Helm Broker. 
 
-Configure the Helm Broker to enable the **Google Cloud Platform Service Broker Provider** class:
-```bash
-kubectl set env -n kyma-system deployment/core-helm-broker -e APP_REPOSITORY_URLS="https://github.com/kyma-project/bundles/releases/download/latest/index.yaml"
```
Comment (klaudiagrz) [inline docs/service-brokers/docs/003-overview-gcp-broker.md]: Shouln't only the URL be updated into: https://github.com/kyma-project/bundles/releases/download/0.2.0/ ?

Comment (piotrmiskiewicz) [reply docs/service-brokers/docs/003-overview-gcp-broker.md]: Goggle Cloud Platform Broker Provider is in the latest and 0.2.0 bundle versions. The default version of bundles used in Kyma contains GCP Broker Provider so there is no need to have such step (which was removed by me).

**Pull request merged**